### DISCUSSION TITLE
Why does Appwrite take up so much disk space?

### DISCUSSION THREAD

**Discussion by Gishan86** [appwrite/appwrite#3739] [Q&A] [answered]
I really love the idea of having a centralized backend for my projects during development.
Setup was fairly easy but I'm a little shocked by the disk space it uses... all in all 40GB.

The space seems to be used by the docker images:
/var/lib/docker/vfs/dir contains multiple 430MB files, many of them with a '-init' suffix.

Isn't that a bit much?
I was expecting maybe 1-2GB like my other containers.
Is this expected or did I do something wrong?

**Comment by eldadfux** [comment]
Hey, what version of Appwrite are you using? We did saw some issues with long running instances where Docker logs were getting really big. In latest version we added a log size limit in the compose spec that should solve this issue.

This is the exact fix we implemented: https://github.com/appwrite/appwrite/blob/master/docker-compose.yml#L6-L11

**Comment by Gishan86** [comment, marked as answer]
I'm running v:0.15.3.402.
I've set it up maybe 2 weeks ago on a new LXC with Debian 11 and the pre-reqisites installed.
During setup it quickly went up to 40GB disk usage and stayed there. Like I said - without any projects/databases created in Appwrite.

I normally don't use docker but wanted to give Appwrite a shot as it seems really nice for my usecase.
But having to backup 40GB for the bare system alone is a bit much...

**Reply by stnguyen90** [reply]
@Gishan86, how did you install Appwrite?

**Reply by stnguyen90** [reply]
Actually, I found [this](https://github.com/mailcow/mailcow-dockerized/issues/840) issue that might be related. In short, LXC might be using vfs as the storage driver rather than overlay2 and the downside of using vfs is docker images take up a lot more space on disk.

**Reply by stnguyen90** [reply]
Circling back on this, the problem is the `vfs` storage driver, so if you [change the storage driver to `overlay2`](https://docs.docker.com/storage/storagedriver/overlayfs-driver/#configure-docker-with-the-overlay-or-overlay2-storage-driver) the images won't take up as much space.

**Reply by parito-ltd** [reply]
@stnguyen90 , so what is actually the expected disk usage by an empty newly configured docker instance of Appwrite when using overlay2? Thanks !
